core/new-gui/src/environments/environment.default.ts

```diff
@@ -45,6 +45,13 @@ export const defaultEnvironment = {
    */
   linkBreakpointEnabled: true,
 
+  /**
+   * the access code for mapbox
+   */
+  mapbox: {
+    accessToken: 'pk.eyJ1IjoiamVyZW15bGkiLCJhIjoiY2lrZ2U4MWI4MDA4bHVjajc1am1weTM2aSJ9.JHiBmawEKGsn3jiRK_d0Gw'
```

Comment:
is the accessToken sensitive at all? if so we should not commit it. if not we can keep it like this.

Reply:
umm not sensitive but if not included, none of the library's functions would work, how can we hide it?

Reply:
If we need to hide it, do not include it in the PR and we can supply the token at deployment.

Reply:
done

Reply:
please remove the token in the final version before merging into master, thanks